### DISCUSSION TITLE
New ​Feature Request: Note Focus View

### DISCUSSION THREAD

**Discussion by houshasen** [toeverything/AFFiNE#6990] [Ideas]
### Description

I truly appreciate how AFFiNE handles notes. While it prevents me from creating cluttered notes, it still allows for bottom-up writing. To enhance this workflow, I would like to request a "Note Focus View."

Essentially, I want the ability to view a selected note as a "page view-like" in conjunction with the edgeless view. I understand this functionality would come after highly anticipated the "Split View". I am not sure if coding for this would be easier, similar or harder than "[concurrent page and edgeless view](https://github.com/toeverything/AFFiNE/issues/1759)".

However, it differs from above slightly because, in this case, the "page" side would only display the selected note portion, which may not always be shown in the Page view. It would be even nicer if we could trigger this feature directly from the Page view as well. Specifically, by selecting a [note block-like](https://github.com/toeverything/AFFiNE/issues/6898) section, we could activate this view mode.

### Use case

As mentioned above, this approach allows us to fully replicate an atomic note writing within document yet not making the clutter.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by CatsJuice** [comment]
Sorry, I don’t quite understand what you mean. Could you explain it more clearly? It looks like you want to control which blocks should be displayed in page mode.

However, currently, our entire page will be inserted into Edgeless as a note. If you want to split it into multiple parts, you actually need to use the “Cut” function in Edgeless (not sure if you are familiar with this function).
![CleanShot 2024-05-16 at 10 31 51@2x](https://github.com/toeverything/AFFiNE/assets/39363750/a58beb2b-9ab4-4626-b339-6e63c33aef75)

**Comment by houshasen** [comment]
Thank you for commenting. 

In short, I want Note in Edgeless mode to get "full" attention i.e. Page Mode-like view. 

For example, I have notes like below.

![Image](https://github.com/toeverything/AFFiNE/assets/48835895/a3b2fc7b-95ef-4de7-92aa-3c64f4629355)

I write individual pieces one at a time. Currently, I draft my notes in Edgeless Mode then combine it to produce final form of writeup. If the note is short enough, I can mag it up and write within Edgeless Mode. However, if a note becomes too long, this mode becomes problematic, and I need to switch to Page Mode, but in Page Mode, I can see all other notes unless I turn them off. Since I use headers to indicate sections, they aren't necessary collapsible.

So rather than doing these back and forth, I am hoping to be able to just simply click on a note and choose "Note Mode" to just focus on the Note, which is essentially Page Mode without any other piece but the selected Note. 

Does this make sense?
